.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,408 @@
+            const status = "${{ steps.summary.outputs.status }}";
+            const summary = "${{ steps.summary.outputs.summary }}";
+            
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `### Smoke Test Gating\n\n${summary}\n\n[View detailed results](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})`
+            });
+
+      - name: Notify Slack on failure
+        if: failure() || needs.smoke-test.result != 'success'
```

actionlint (1.7.12)

[error] property "smoke-test" is not defined in object type {deploy: {outputs: {deployment_id: string; staging_url: string}; result: string}}

(expression)